internal/exporter/pollingexporter/README.md

```diff
@@ -0,0 +1,28 @@
+# Polling Exporter
+
+This exporter makes span data available via a HTTP endpoint. The endpoint
+accepts requests for spans with specific characteristics and blocks until the exporter
+receives such spans or the request times out. Once the requested spans are detected, they're
+returned back to the client as JSON. 
+
+This exporter return data as [JSON encoding](https://developers.google.com/protocol-buffers/docs/proto3#json)
```

Comment:
```suggestion
This exporter returns data as [JSON encoding](https://developers.google.com/protocol-buffers/docs/proto3#json)
```

It would be helpful to show an example request w/ params and the response imo.

Reply:
Updated description with link: https://www.loom.com/share/16256b778d9a40a297a2212692422d0b